crates/handler/src/websocket/controller.rs

```diff
@@ -154,11 +155,14 @@ impl WebSocketContext {
             true => "wss",
             false => "ws",
         };
-        let url = match &route.query_path {
+        // dbg!(route);
+        let url = match &route.subscribe_path {
```

Comment:
@sunli829  this is where i've made the change to use the current subscribe path, so i think maybe this should default to the query path as it is currently or we should have a separate websocket path. So the config might look like: 

[[services]]
name = "ruby-api"               
addr = "localhost:9000"       
query_path = "/graphql"               
subscribe_path = "/subscription" 
websocket_path = "/graphql-ws"           
introspection_path = "/sdl"    

Leaves the user the option to allow different paths for both subscriptions and those using the grahpql-transport-ws protocol.

Reply:
This looks great! 😁